### DISCUSSION TITLE
[Unstable] RuinConfig should be changed to always have at least one RuinEntrance

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14782] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Now you need to find entrance to get into the ruin since ruin wall are indestructible now
Although many ruin modules have new weakened walls or potential entrance even if they are not entrance module
but you can still encounter a ruin that has no entrance if you are unlucky I guess?
Ruin is not guaranteed to have RuinEntrance module

### Reproduction steps

1. Check RuinConfig.xml
2. Some RuinConfig do not have a entrance module

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in the update, closing
